cfme/tests/physical_infrastructure/api/test_physical_switch_api.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.physical.provider.lenovo import LenovoProvider
```

Comment:
empty line here, sorry to nitpick but until we get `black` enabled across the repo we'll enforce stuff like this as much as possible.